test/specs/collections/Breadcrumb/Breadcrumb-test.js

```diff
@@ -12,14 +12,29 @@ describe('Breadcrumb', () => {
       .should.have.tagName('div')
   })
 
+  const sections = [
+    { text: 'Home', link: true },
+    { text: 'T-Shirt', href: 'google.com' },
+  ]
+
   it('renders children with `sections` prop', () => {
-    const sections = [
-      { text: 'Home', link: true },
-      { text: 'T-Shirt', href: 'google.com' },
-    ]
     const wrapper = shallow(<Breadcrumb sections={sections} />)
 
     wrapper.find('BreadcrumbDivider').should.have.length(1)
     wrapper.find('BreadcrumbSection').should.have.length(2)
   })
+
+  it('renders defined divider with `divider` prop', () => {
+    const wrapper = mount(<Breadcrumb sections={sections} divider='>' />)
+    const divider = wrapper.find('BreadcrumbDivider').first()
+
+    divider.text().should.to.equal('>')
```

Comment:
We use the [chai-enzyme](https://github.com/producthunt/chai-enzyme) plugin.  This allows us to move the [enzyme](https://github.com/airbnb/enzyme/) method to the end of the assertion chain:

```diff
-divider.text().should.to.equal('>')
+divider.should.contain.text('>')
```

This makes for much more readable errors as well.  Compare the assertion syntax here with the error messages:

```jsx
shallow(<Button>foo</Button>)
  .text().should.equal('bar')

//=> expected 'foo' to equal 'bar'
```

```jsx
shallow(<Button>foo</Button>)
  .should.contain.text('bar')

//=>  expected <Button /> to contain text 'bar', but it has 'foo'      
         
         HTML:
         
         <button type="button" class="ui button">foo</button>
```